common/propsValues.ts

```diff
@@ -0,0 +1,38 @@
+import {collapseVerticalTransition, fadeTransition} from '../core/lib/transitions/bootstrap';
+import {getPaginationDefaultConfig, getRatingDefaultConfig} from '@agnos-ui/core';
+
+export interface SelectValues {
+	value: string;
+	label: string;
+}
+
+const propsValuesList: Record<string, any> = {
+	//common
+	log: (x: any) => console.log(x),
+	noop: () => {},
+	//transitions
+	collapse: collapseVerticalTransition,
+	fade: fadeTransition,
+	//rating
+	ratingValue: getRatingDefaultConfig().ariaValueTextFn,
+	hartsRatingValue: (rating: number, maxRating: number) =>
+		`${rating} ${rating === 1 ? 'heart' : 'hearts'} out of ${maxRating} ${maxRating === 1 ? 'heart' : 'hearts'}`,
+	startRatingValue: (rating: number, maxRating: number) =>
+		`${rating} ${rating === 1 ? 'star' : 'stars'} out of ${maxRating} ${maxRating === 1 ? 'star' : 'stars'}`,
+	starIconRating: getRatingDefaultConfig().slotStar,
+	heartIconRating: ({fill}: {fill: number}) => String.fromCharCode(fill === 100 ? 9829 : 9825),
+	//pagination
+	defaultPageFactory: getPaginationDefaultConfig().pagesFactory,
+	simpleLabel: getPaginationDefaultConfig().ariaPageLabel,
+	pageLabel: (processPage: number, pageCount: number) => `Page ${processPage} of ${pageCount}`,
+	slotNumberLabel: getPaginationDefaultConfig().slotNumberLabel,
+};
+
+export const getPropValues = (config: Record<string, any>) => {
```

Comment:
I think `getPropValues` should not modify its argument. It is better to do a copy before doing the change.
Otherwise, it should be clear in its name that it mutates the argument, but then we should not call it with something we directly got from dependency injection.